presto-main/src/main/java/io/prestosql/type/JsonPathType.java

```diff
@@ -44,18 +42,29 @@ public Object getObjectValue(ConnectorSession session, Block block, int position
     @Override
     public void appendTo(Block block, int position, BlockBuilder blockBuilder)
     {
-        throw new UnsupportedOperationException();
+        if (block.isNull(position)) {
```

Comment:
Why does this type need this method but the others do not?

Reply:
It doesn't